jenkins/jobs/ci2/pingcap/tiflow/dm_ghpr_compatibility_test.groovy

```diff
@@ -23,12 +23,13 @@ pipelineJob('dm_ghpr_compatibility_test') {
                     whitelist("")
                     orgslist("pingcap")
                     whiteListTargetBranches {
-                        ghprbBranch { branch('^(release-)?[5]\\.[3-4](\\.\\d+)?(\\-.*)?$') }
                         ghprbBranch { branch('^(release-)?[6]\\.[0-9](\\.\\d+)?(\\-.*)?$') }
                         ghprbBranch { branch('^(release-)?[7]\\.0(\\.\\d+)?(\\-.*)?$') }
                     }
                     blackListTargetBranches {
                         ghprbBranch { branch('master') }
+                        ghprbBranch { branch('^(release-)?5\\.[3-4]\\d*(\\.\\d+)?(\\-.*)?$') }
+                        ghprbBranch { branch('^(release-)?6\\.[1|5]\\d*(\\.\\d+)?(\\-.*)?$') }
```

Comment:
missing line for `7`?

Reply:
It is not necessary when`whiteListTargetBranches` has already been configured.